.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
     "files.associations": {
         "**/eng/pipelines/*.yml": "azure-pipelines"
```

Comment:
I'm gonna be honest. we better not have yaml files being used for anything else in this repo so I am tempted to say this should just be `*.yml`

Reply:
but we should only have one conceptual change per PR. Don't want to go crazy